### PR TITLE
Add joint tables and new queries

### PR DESCRIPTION
**Pull request by devalibello**
This pull request uses many-to-many joined tables and adds new queries to the database. The result of the requires are provided in the screenshots below.

    - Who was the last animal seen by William Tatcher?

    - How many different animals did Stephanie Mendez see?

    - List all vets and their specialties, including vets with no specialties.

    - List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.

    - What animal has the most visits to vets?

    - Who was Maisy Smith's first visit?

    - Details for most recent visit: animal information, vet information, and date of visit.

    - How many visits were with a vet that did not specialize in that animal's species?

    - What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

**Comment by binhussen** [comment]
@devalibello kindly would remove the `live demo` section and `note` below the `mit` to make your readme professional

**Comment by devalibello** [comment]
> @devalibello kindly would remove the `live demo` section and `note` below the `mit` to make your readme professional

Noted @binhussen

**Comment by devalibello** [comment]
@binhussen The readme file has been updated. Thanks 😄

**Comment by binhussen** [comment]
@devalibello great job, please delete the above `Noted @binhussen` comment

File: data.sql [hunk 1]
```diff
@@ -150,4 +150,100 @@ WHERE name = 'Devimon' OR name = 'Plantmon';
...
+-- Query 9
+SELECT species.name, COUNT(*)
+FROM visits
+JOIN animals ON visits.animal_id = animals.id
+JOIN species ON animals.species_id = species.id
+JOIN vets ON vets.id = visits.vet_id
+WHERE vets.name = 'Maisy Smith'
+GROUP BY species.name
+ORDER BY COUNT(*) DESC
+LIMIT 1;
```
Comment (binhussen) [inline data.sql]: - Kindly, everything that is a query that helps to analyze data should be saved in the `queries.sql` file to make your query clean 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @devalibello , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Added "join table" for visits✔️
- Well Structured Query ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @devalibello 

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Join table is added, perfect!
-  ✅ Great work using the right GitFlow 
-  ✅ Your README looks good, well done!

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**